src/composition/watchers/composition.rs

```diff
@@ -123,6 +123,27 @@ where
                                     .tap_err(|err| error!("{:?}", err));
                             };
                         }
+                        Subgraph(SubgraphEvent::RoutingUrlChanged(routing_url_changed)) => {
+                            info!("Change of routing_url detected for subgraph '{}'", routing_url_changed.name());
+                            let name = routing_url_changed.name().clone();
+                            match self.supergraph_config.get_subgraph(&name) {
+                                None => {
+                                    debug!("Could not find subgraph '{}'", name);
+                                    continue
+                                },
```

Comment:
This arm is really only included for completeness, as if this method is called there should be a subgraph otherwise it wouldn't be being called in the first place.